Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1333,6 +1341,8 @@ void MediaPlayerPrivateGStreamer::handleMessage(GstMessage* message)
 
         m_errorMessage = err->message;
         error = MediaPlayer::Empty;
+        m_odhReporter.report(ODH_REPORT_AVPIPELINE_STATE_PLAYBACK_ERROR, m_errorMessage.utf8().data(), OdhMediaType::VIDEO, m_avContextGetter);
```

Comment:
cannot we support simpler interface?
m_odhReporter.report(ODH_REPORT_AVPIPELINE_STATE_PLAYBACK_ERROR, m_errorMessage.utf8().data(), OdhMediaType::VIDEO && OdhMediaType::AUDIO, m_avContextGetter);